src/new-client/src/Web/WebApp.tsx

```diff
@@ -29,6 +37,36 @@ export const WebApp: React.FC = () => (
           )
         }}
       />
+      <Route
+        path="/liverates"
+        render={() => {
+          return (
+            <TornOutTileContext.Provider value={{ isTornOut: true }}>
+              <LiveRates />{" "}
+            </TornOutTileContext.Provider>
+          )
+        }}
+      />
+      <Route
+        path="/trades"
+        render={() => {
+          return (
+            <TornOutTradeContext.Provider value={{ isTornOut: true }}>
```

Comment:
See comment about single context. If you use a single context then you could wrap all these routes as children of the context, reducing the duplicaton.

Reply:
Done.